qa/vagrant/src/test/resources/packaging/scripts/70_sysv_initd.bats

```diff
@@ -117,6 +117,23 @@ setup() {
     [ "$status" -eq 3 ] || [ "$status" -eq 4 ]
 }
 
+@test "[INIT.D] don't mkdir when it contains a comma" {
+    # Remove these just in case they exist beforehand
+    rm -rf /tmp/aoeu,/tmp/asdf
+    rm -rf /tmp/aoeu,
+    # set DATA_DIR to DATA_DIR=/tmp/aoeu,/tmp/asdf
+    sed -i 's/DATA_DIR=.*/DATA_DIR=\/tmp\/aoeu,\/tmp\/asdf/' /etc/init.d/elasticsearch
+    cat /etc/init.d/elasticsearch | grep "DATA_DIR"
+    service elasticsearch start
+    wait_for_elasticsearch_status
+    assert_file_not_exist /tmp/aoeu,/tmp/asdf
```

Comment:
I'm not sure if `mkdir -p /tmp/aoeu,/tmp/asdf` will be interpreted as making a directory named `/tmp/aoeu,/tmp/asdf`. I think that because of that third `/` it will appear to `mkdir -p` as requesting a tree like `/tmp -> /aoeu, -> /tmp -> /asdf`.